jfrog-token/main.test.ts

```diff
@@ -32,10 +34,112 @@ describe("jfrog-token", async () => {
     port: 0,
   });
 
-  testRequiredVariables(import.meta.dir, {
-    agent_id: "some-agent-id",
-    jfrog_url: "http://" + fakeFrogHost.hostname + ":" + fakeFrogHost.port,
-    artifactory_access_token: "XXXX",
-    package_managers: "{}",
+  const fakeFrogHostAndPort = `${fakeFrogHost.hostname}:${fakeFrogHost.port}`;
+  const fakeFrogUrl = `http://${fakeFrogHostAndPort}`;
+
+  it("can run apply with required variables", async () => {
+    testRequiredVariables(import.meta.dir, {
```

Comment:
Same comment as the auth test file. Looks like the type can be set up like this?
```ts
type TestVariables = Readonly<{
  agent_id: string;
  jfrog_url: string;
  artifactory_access_token: string;

  jfrog_server_id?: string;
  token_description?: string;
  check_license?: boolean;
  refreshable?: boolean;
  expires_in?: number;
  configure_code_server?: boolean;
  package_managers?: string;
}>;
```